sphinxcontrib/images.py

```diff
@@ -151,7 +154,7 @@ def run(self):
         #TODO: something is broken here, not parsed as expected
         description = nodes.paragraph()
         content = nodes.paragraph()
-        content += [nodes.Text(u"%s" % x) for x in self.content]
+        content += [nodes.Text("%s" % x) for x in self.content]
```

Comment:
same here ^^

Reply:
So we could keep the ``u"string"`` style for compatibility, is that your thought?